PWGCF/Flow/Tasks/FlowGFWPbPb.cxx

```diff
@@ -46,14 +52,15 @@ using namespace o2::aod::evsel;
 
 #define O2_DEFINE_CONFIGURABLE(NAME, TYPE, DEFAULT, HELP) Configurable<TYPE> NAME{#NAME, DEFAULT, HELP};
 
-struct FlowGFWPbPb {
+struct flowGFWPbPb {
```

Comment:
If you modify your source file according with the `linter` you should use the same name but with the first letter capitalized for your task name